ui/components/app/edit-gas-fee-popover/network-statistics/network-statistics.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import {
+  COLORS,
+  TYPOGRAPHY,
+} from '../../../../helpers/constants/design-system';
+import { useGasFeeContext } from '../../../../contexts/gasFee';
+import I18nValue from '../../../ui/i18n-value';
+import Typography from '../../../ui/typography/typography';
+
+import { BaseFeeTooltip, PriorityFeeTooltip } from './tooltips';
+import StatusSlider from './status-slider';
+
+const NetworkStatistics = () => {
+  const { gasFeeEstimates } = useGasFeeContext();
+
+  return (
+    <div className="network-statistics">
+      <Typography
+        color={COLORS.UI4}
+        fontWeight="bold"
+        margin={[3, 0]}
+        tag={TYPOGRAPHY.H6}
+        variant={TYPOGRAPHY.H8}
```

Comment:
Just curious why would we want the `tag` and `variant` to be different?

Reply:
also fixed to use TYPOGRAPHY.H8